package/Codegen/TSTypes.cs

```diff
@@ -173,218 +173,218 @@ public virtual string TransformText()
             
             #line default
             #line hidden
             this.Write(") => {\r\n  const tag = findNodeHandle(ref.current);\r\n  UIManager.dispatchViewManag" +
                     "erCommand(tag, UIManager.getViewManagerConfig(\'XamlControl\').Commands.");
             
-            #line 67 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
+            #line 65 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
             this.Write(this.ToStringHelper.ToStringWithCulture(command.Name));
             
             #line default
             #line hidden
             this.Write(", ");
             
-            #line 67 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
+            #line 65 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
             this.Write(this.ToStringHelper.ToStringWithCulture(command.TSArgTypes != null ? "[args]" : "args"));
             
             #line default
             #line hidden
             this.Write(");\r\n};\r\n");
             
-            #line 69 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
+            #line 67 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
   } 
             
             #line default
             #line hidden
             this.Write("export const ");
             
-            #line 70 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
+            #line 68 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
             this.Write(this.ToStringHelper.ToStringWithCulture(Util.ToJsName(type)));
             
             #line default
             #line hidden
             this.Write(" = (");
             
-            #line 70 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
-            this.Write(this.ToStringHelper.ToStringWithCulture(BaseTypeName));
+            #line 68 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
+            this.Write(this.ToStringHelper.ToStringWithCulture(PartialTypeName));
             
             #line default
             #line hidden
             this.Write(" as (ForwardRefExoticComponent<React.PropsWithChildren<");
             
-            #line 70 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
+            #line 68 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
             this.Write(this.ToStringHelper.ToStringWithCulture(PropsTypeName));
             
             #line default
             #line hidden
             this.Write("> & RefAttributes<");
             
-            #line 70 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
+            #line 68 "C:\Repos\react-native-xaml-mine\package\Codegen\TSTypes.tt"
             this.Write(this.ToStringHelper.ToStringWithCulture(RefTypeName));
             
             #line default
             #line hidden
             this.Write(">> ");
```

Comment:
nit: NativeMethodsFor already starts with a space. I think you can remove this one?